dashboard/models.py

```diff
@@ -168,6 +168,9 @@ class Course(models.Model):
     date_end = models.DateTimeField(verbose_name="End Date and Time", null=True, blank=True)
     show_grade_counts = models.BooleanField(blank=False, null=False, default=False, verbose_name=
                                          "Show Grade Counts")
+    GRADING_CHOICES = [('Percent', 'Percent'), ('Point', 'Point'), ]
+    myla_grading_type = models.CharField(verbose_name="Grading Type", max_length=255,
```

Comment:
instead of `myla_grading_type` change it to `show_grade_type` and change the verbose name to `Show Grade Type'

since this is a model change, delete the migration file `0013_course_myla_grading_type.py` and run the makemigration again and commit